libshaderc/include/shaderc.h

```diff
@@ -105,6 +111,11 @@ void shaderc_compile_options_release(shaderc_compile_options_t options);
 void shaderc_compile_options_add_macro_definition(
     shaderc_compile_options_t options, const char* name, const char* value);
 
+// Set flags for the rules on which warnings or errors will be issued. This can
+// either be an individual flag, or a set of flags or'd together
```

Comment:
That's actually following the precise pattern used by glslang and EShMessages which this enum maps to. See glslang/Public/ShaderLang.h line 127 and then subsequent uses. I do agree it's unusual, but my thinking was that it'd be best to keep it close to the underlying options we're exposing from glslang.